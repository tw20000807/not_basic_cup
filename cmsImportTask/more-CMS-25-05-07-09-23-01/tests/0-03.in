xBKhCirDv7dWDvJlxKTtuZimwF4Zi9x1
1
9 1
957615591 539007519 328490334 1307233671 1310738644 1449642802 1289279528 717260556 1491658535
1841871307 1310738644 328490334 717260556 1307233671 957615591 539007519 1449642802 1289279528 1491658535
