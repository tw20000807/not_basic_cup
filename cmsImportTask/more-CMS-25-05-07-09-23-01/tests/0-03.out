du3W59OSqKqC9FuBgxi0VkGO13qNOHq6
1841871307
